<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="log_Browser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>501</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="parse_image">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>开始分包</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_log">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除log</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>711</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="choose_ap">
       <property name="text">
        <string>选择AP目录</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="show_ap"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="choose_out">
       <property name="text">
        <string>选择输出目录</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="show_out"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="choose_xml">
       <property name="text">
        <string>选择XML</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="show_xml"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
